saw-core/src/SAWCore/SCTypeCheck.hs

```diff
@@ -457,7 +455,7 @@ instance TypeInfer (FlatTermF Term) where
   typeInferComplete ftf =
     SCTypedTerm
     <$> liftTCM scFlatTermF ftf
-    <*> withErrorTermF (FTermF ftf) (typeInfer ftf)
+    <*> {- withErrorTermF (FTermF ftf) -} (typeInfer ftf)
```

Comment:
Should this commented-out code be removed? Or is it useful to keep it around for debugging purposes?

Reply:
It should have been removed; fixed in 583307a11205ad7ee16e13dcb48aba1c47d6345b.